docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx
+sphinx-markdown-tables
+recommonmark
+sphinx-rtd-theme
```

Comment:
Is this file needed if we have the packages listed in `pyproject.toml`?

Reply:
It's needed for readthedocs unfortunately.